### PR TITLE
signUp

### PR DESCRIPTION
**Pull request by imsandip2085**

### CODE REVIEW

File: polling-app/src/Components/Dashboard/dashBoard.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { connect } from "react-redux";
+import {Link} from 'react-router-dom';
+
+class DashBoard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  componentDidMount = () => {
+    this.props.GetPollStatus();
```
Comment (nodeexcel) [inline polling-app/src/Components/Dashboard/dashBoard.js]: always use proper variable name

File: polling-app/src/Components/Dashboard/dashBoard.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        {this.props.getPollStatus.response &&
+          this.props.getPollStatus.response.length &&
+          this.props.getPollStatus.response.map((val, key) => {
+            return (
+              <Card style={{ margin: "40px 60px" }} className={"card"}>
+                <Card.Body>
+                  <Card.Title>Title : {val.title} </Card.Title>
+                  Options :
+                  <ul>
+                    {val.options.map(res => {
```
Comment (nodeexcel) [inline polling-app/src/Components/Dashboard/dashBoard.js:40]: why code is not formatted

File: polling-app/src/Components/Login/loginForm.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+      this.setState({ username: "" });
+      this.setState({ password: "" });
+    }
+  };
+
+  render() {
+    if (this.props.LoginStatus && this.props.LoginStatus.isLogin) {
+      this.props.history.push("/dashboard");
+    }
+    console.log(this.props.LoginStatus);
```
Comment (nodeexcel) [inline polling-app/src/Components/Login/loginForm.js]: never log come in pr

File: polling-app/src/Components/Dashboard/dashBoard.js (hunk 1 above)
Comment (imsandip2085) [reply polling-app/src/Components/Dashboard/dashBoard.js]: ok sir

File: polling-app/src/Components/Dashboard/dashBoard.js (hunk 2 above)
Comment (imsandip2085) [reply polling-app/src/Components/Dashboard/dashBoard.js:40]: sorry sir, I forget

File: polling-app/src/Components/Login/loginForm.js (hunk 3 above)
Comment (imsandip2085) [reply polling-app/src/Components/Login/loginForm.js]: ok sir

File: polling-app/src/Components/Registation/registationForm.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    if (this.props.registerStatus && this.props.registerStatus.isRegistion) {
+      this.setState({ username: "" });
+      this.setState({ password: "" });
+    }
+  };
+  render() {
+    if (this.props.registerStatus && this.props.registerStatus.isRegistion) {
+      this.props.history.push("/login");
+    }
+    if (
```
Comment (Rahul-Khera-Codes) [inline polling-app/src/Components/Registation/registationForm.js]: what is this for

File: polling-app/src/service/apicall.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+        response = await axios.get(url, { header: header });
+        return response.data;
+      case "post":
+        response = await axios.post(url, { headers: header, data: data });
+        return response.data;
+      default:
+        break;
+    }
+  } catch (error) {
+    console.error(error);
```
Comment (Rahul-Khera-Codes) [inline polling-app/src/service/apicall.js]: return error from here

File: polling-app/src/testpage.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+          <h1>Take Poll</h1>
+        </div>
+        <Card
+          style={{ width: "40rem", margin: "40px 120px" }}
+          className={"card"}
+        >
+          <Card.Body>
+            <Card.Title>Title : </Card.Title>
+            Options :
+            <ul>
```
Comment (Rahul-Khera-Codes) [inline polling-app/src/testpage.js]: why this hard coded

File: polling-app/src/Components/Registation/registationForm.js (hunk 4 above)
Comment (imsandip2085) [reply polling-app/src/Components/Registation/registationForm.js]: if registration is completed then push on login pages and line 40-44 have no meaning I will remove it

File: polling-app/src/service/apicall.js (hunk 5 above)
Comment (imsandip2085) [reply polling-app/src/service/apicall.js]: ok sir

File: polling-app/src/testpage.js (hunk 6 above)
Comment (imsandip2085) [reply polling-app/src/testpage.js]: ok sir i will give proper value

**Pull request merged**